vcd/provider.go

```diff
@@ -95,6 +95,7 @@ func Provider() terraform.ResourceProvider {
 			"vcd_edgegateway_vpn":  resourceVcdEdgeGatewayVpn(),
 			"vcd_vapp_vm":          resourceVcdVAppVm(),
 			"vcd_org":              resourceOrg(),
+			"vcd_vdc":              resourceVcdVdc(),
```

Comment:
Should we call it "vcd_org_vdc" to avoid future misunderstandings?

Reply:
Renamed.